### PR TITLE
created get table cards use case

### PR DESCRIPTION
**Pull request by danieljimenezr**

### CODE REVIEW

**Comment by rgomezcasas** [comment]
You've commited the `.idea` folder U_U

**Comment by GMadorell** [comment]
yeah, put the .idea folder in your global .gitignore :P

File: src/main/scala/com/lambtors/poker_api/module/poker/application/table/find/FindTableCardsQueryHandler.scala [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.lambtors.poker_api.module.poker.application.table.find
+
+import com.lambtors.poker_api.infrastructure.query_bus.QueryHandler
+import com.lambtors.poker_api.module.poker.domain.model.{Card, GameId}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+final class FindTableCardsQueryHandler(tableCardsFinder: TableCardsFinder)
+                                      (implicit ec: ExecutionContext)
+  extends QueryHandler[FindTableCardsQuery, List[Card]] {
```
Comment (GMadorell) [inline src/main/scala/com/lambtors/poker_api/module/poker/application/table/find/FindTableCardsQueryHandler.scala:9]: return a response?

File: src/main/scala/com/lambtors/poker_api/module/poker/domain/model/CardValue.scala [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+case object Ace extends CardValue("A")
+case object Two extends CardValue("2")
+case object Three extends CardValue("3")
+case object Four extends CardValue("4")
+case object Five extends CardValue("5")
+case object Six extends CardValue("6")
+case object Seven extends CardValue("7")
+case object Eight extends CardValue("8")
+case object Nine extends CardValue("9")
+case object Jay extends CardValue("J")
```
Comment (GMadorell) [inline src/main/scala/com/lambtors/poker_api/module/poker/domain/model/CardValue.scala]: we're missing the 10

File: src/main/scala/com/lambtors/poker_api/module/poker/application/table/find/TableCardsFinder.scala [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+import scala.concurrent.{ExecutionContext, Future}
+
+final class TableCardsFinder(repository: PokerGameRepository)
+                            (implicit ec: ExecutionContext) {
+  def find(gameId: GameId): Future[List[Card]] = repository.search(gameId).map(
+    gameOpt => if (gameOpt.isDefined) {
+      gameOpt.get.tableCards
+    } else {
+      throw PokerGameNotFound(gameId)
```
Comment (GMadorell) [inline src/main/scala/com/lambtors/poker_api/module/poker/application/table/find/TableCardsFinder.scala]: put this inside a future failed? we should probably never naked throw an exception

Comment (danieljimenezr) [reply src/main/scala/com/lambtors/poker_api/module/poker/application/table/find/TableCardsFinder.scala]: this is the imperative mode! when you throw an exception inside future, it is a future failed

**Comment by danieljimenezr** [comment]
test will be done later, we have to do other use cases

**Pull request merged**